plugins/inputs/postgresql/postgresql.go

```diff
@@ -42,6 +40,20 @@ var sampleConfig = `
   ##
   address = "host=localhost user=postgres sslmode=disable"
 
+  ## connection configuration.
+
+  ## maxidle - specify the number of idle connections to maintain. see sql.DB for details.
+  ## default is 1. setting to 0 to force disconnect after use.
+  max_idle = 1
```

Comment:
I don't think this feature should be added. Because of how the plugin works, after the polling interval is complete, every single connection is going to be idle. Thus it's going to result in connections constantly being created and destroyed, defeating the purpose of the persistence.